main.go

```diff
@@ -15,8 +15,13 @@ limitations under the License.
 */
 package main
 
-import "github.com/strangelove-ventures/lens/cmd"
+import (
+	"log"
+
+	"github.com/strangelove-ventures/lens/cmd"
+)
 
 func main() {
+	log.SetFlags(log.LstdFlags | log.Lshortfile)
```

Comment:
oh what is this!?

Reply:
oops, forgot to delete it, I wanted to see filenames in logs/errors. it turned out to not be useful. I'll remove it

Reply:
Oh cool, just never seen that annotation before!